### ISSUE TITLE
Allow accelerator to be bind with mouse input

### ISSUE THREAD

**Issue by bub-bl** [Facepunch/sbox-issues#6524]
### For?

S&Box

### What can't you do?

We can bind accelerator with keyboard input but not with mouse input

### How would you like it to work?

Allow accelerator to bind with mouse input

### What have you tried?

Above

**Comment by handsomematt** [comment]
A what

**Comment by bub-bl** [comment]
> A what

A modifier key if you prefer (like ALT, CTRL, SHIFT)

**Comment by handsomematt** [comment]
I don't see any reason or technical limitations on why we couldn't do this. Didn't we used to have Input.KeyModifier or something though @DevulTj

**Comment by DevulTj** [comment]
Yeah, can't remember why we took it out - part of me wants to just make input actions have a list of buttons so you can combine anything you want really
